test/mechanics/cell/Jacobian.cpp

```diff
@@ -59,6 +61,42 @@ BOOST_AUTO_TEST_CASE(Jacobian1Din2D)
     }
 }
 
+//! @brief calculate the area of a triangle (points a,b,c) element based on ||J|| and check vs. `correctArea`
+void CheckJacobian2Din3D(Eigen::Vector3d a, Eigen::Vector3d b, Eigen::Vector3d c, double correctArea)
```

Comment:
Can we name this `TriangleCheck2Din3D` or something, `CheckJacobian` seems more general than it is.